includes/data-stores/interfaces/wc-order-data-store-interface.php

```diff
@@ -0,0 +1,73 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Order Data Store Interface
+ *
+ * Functions that must be defined by order store classes.
+ *
+ * @version  2.7.0
+ * @category Interface
+ * @author   WooThemes
+ */
+interface WC_Order_Data_Store_Interface {
+
+	/**
+	 * Read order items of a specific type from the database for this order.
+	 *
+	 * @param  WC_Order $order
+	 * @param  string $type
+	 * @return array
+	 */
+	public function read_items( $order, $type );
+
+	/**
+	 * Remove all line items (products, coupons, shipping, taxes) from the order.
+	 *
+	 * @param WC_Order
+	 * @param string $type Order item type. Default null.
+	 */
+	public function delete_items( $order, $type = null );
```

Comment:
If the classes implement both interfaces, do we need to repeat read_items, delete_items, etc? Or should we just include the additional functions like get_total_refunded.